code/modules/trade/datums/_trade_station.dm

```diff
@@ -1,6 +1,9 @@
-#define good_data(nam, rand_list) list("name" = nam, "amount_range" = rand_list)
+#define good_data(nam, randList) list("name" = nam, "amount_range" = randList)
```

Comment:
We should be using snake_case over CamelCase, it's in our contributing guidelines alongside /tg/ station, and other servers.